content/news/2022-04-12-bevy-0.7/index.md

```diff
@@ -0,0 +1,383 @@
+* GLTF animation importing
+* Unlimited* point lights in a scene
+* Improved clustered forward rendering: dynamic/adaptive clustering, faster and more accurate cluster assignment
+* Compressed texture support (KTX2 / DDS / .basis): load more textures in a scene, faster
+* Compute shader / pipeline specialization: Bevy's flexible shader system was ported to compute shaders, enabling hot reloading, shader defs, and shader imports
+* Render to texture: cameras can now be configured to render to a texture instead of a window
+* Flexible user-customizable mesh vertex layouts in shaders
+* ECS improvements: Order systems using their names, Query::get_many, use conflicting parameters in systems via ParamSets, WorldQuery derives
+* More audio control: pause, volume, speed, and looping
+* Hot reloading for "plugin-provided / built-in assets"
+* Power usage options to enable only updating Bevy Apps when input occurs 
+
+<!-- more -->
+
+## Skeletal Animation
+
+<div class="release-feature-authors">authors: @james7132, @mockersf, @lassade, @Looooong</div>
+
+Bevy finally supports 3D skeletal animation!
+
+<video controls loop><source  src="skeletal_animation.mp4" type="video/mp4"/></video>
+
+<div style="font-size: 1.0rem" class="release-feature-authors">Scene Credits: <a href="https://skfb.ly/6TsvL">Tanabata evening - Kyoto inspired city scene</a> by Mathias Tossens is licensed under <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution</a>. Character model and animation are royalty free assets from Mixamo. 
+</div>
+
+Skeletal animations can now be played, paused, scrubbed, looped, reversed, and speed controlled using the new [`AnimationPlayer`] component and [`AnimationClip`] asset:
+
+```rust
+#[derive(Component)]
+struct Animations {
+    dance: Handle<AnimationClip>,
+}
+
+fn system(mut query: Query<(&Animations, &mut AnimationPlayer)>) {
+    for (animations, mut animation_player) in query.iter_mut() {
+        animation_player.play(animations.dance.clone());
+    }
+}
+```
+
+[`AnimationPlayer`] can also be used to animate arbitrary [`Transform`] components, not just skeletons!
+
+This critical feature has been a long time coming, but we wanted to build it in a way that meshed nicely with the [new Bevy renderer](/news/bevy-0-6/#the-new-bevy-renderer) and didn't just "hack things in". This builds on our new [User-Customizable Mesh Vertex Layouts](LINKME), [Shader Imports](/news/bevy-0-6/#shader-imports), and [Material](/news/bevy-0-6/#materials) systems, which ensures that this logic is flexible and reusable, even with non-standard meshes and custom render pipelines.
+
+And we're just getting started! Multi-track animation blending and higher level animation state management should arrive in the very near future. Now is a great time to start contributing animation features to Bevy. We've smashed through most of the foundational technical hurdles and what remains is largely high level api design choices!
```

Comment:
It'd be great to refer/link to these RFCs directly. Hint hint nudge nudge wink wink.

Reply:
https://github.com/bevyengine/rfcs/pull/51: Composition

https://github.com/bevyengine/rfcs/pull/49: Primitives

Reply:
Added!